Sources/MongoSwift/Operations/DropIndexesOperation.swift

```diff
@@ -2,14 +2,23 @@ import CLibMongoC
 
 /// Options to use when dropping an index from a `MongoCollection`.
 public struct DropIndexOptions: Encodable {
+    /// An arbitrary BSON type to help trace the index through
+    /// the database profiler, currentOp and logs. The default is to not send a value.
+    public var comment: BSON?
```

Comment:
ditto adding a test for this

Reply:
Added test `testCreateListDropIndexesComment()` in `MongoCollection+IndexTests`